build.md

```diff
@@ -382,6 +382,19 @@ Service builds MAY be granted access to multiple secrets. Long and short syntax
 same Compose file. Defining a secret in the top-level `secrets` MUST NOT imply granting any service build access to it.
 Such grant must be explicit within service specification as [secrets](spec.md#secrets) service element.
 
+### tags
+
+`tags` defines a list of tag mappings that MUST be associated to the build image.
+
+```yml
+tags:
+  - "myimage:mytag"
+  "registry/username/myrepos:my-other-tag"
+```
+
+Compose implementations MUST create matching entry with the IP address and hostname in the container's network
```

Comment:
Was this copied from elsewhere? The sentence seems out of place in this section.

Reply:
Definitely 🤦‍♂️